webapp/src/components/propertyValueElement.tsx

```diff
@@ -44,7 +47,7 @@ const PropertyValueElement = (props:Props): JSX.Element => {
             return emailRegexp.test(val)
         }
         case 'url': {
-            const urlRegexp = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w\-_]*)?\??(?:[-+=&;%@.\w_]*)#?(?:[.!/\\\w]*))?)/
+            const urlRegexp = /((((http|https):(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w\-_]*)?\??(?:[-+=&;%@.\w_]*)#?(?:[.!/\\\w]*))?)/
```

Comment:
I'm ok with this, but I don't know if that is right, a url is a url, and can be anything else, not only http or https. We can decide to not show the link if it is not http or https (something that is also weird to me), but not allow somebody to use a, for example, ftp url here, for me, makes not sense.

Reply:
@jespino Now that you put it that way, I agree with this. IMO we shouldn't even restrict the protocol length between 3 and 9 and allow any arbitrary length. For example, I can create my own desktop app which opens links registered with `foobarbaz://` protocol.

I've created this to weekly agenda, proposing any arbitrary protocol.

Reply:
This is done now.